src/App.js

```diff
@@ -8,6 +8,12 @@ import PdfviewerContainer from './components/PdfviewerContainer';
 import Addgrantframe from './components/Addgrantframe';
 import Topbar from './components/Topbar';
 
+// Only include tour in the browser
+let Tour = <div></div>;
+if (process.env.TEST !== 'true') {
+  Tour = require('./containers/Tour').default;
```

Comment:
Can you mock the `Tour` component in the test instead? I think that's something like 

```
jest.doMock('../containers/Tour', () => (<div />));
```